bin/content-push

```diff
@@ -14,7 +14,7 @@ end
 
 def check_changes
   puts `git status`
-  print "Should I push all of these files? [Y/n] "
+  print "Should I commit and push all of these unstaged files? Untracked files will be left alone [Y/n] "
```

Comment:
are you sure untracked files are left alone? If they are it's a bug in the content push script, they should be added.